packages/office-ui-fabric-react/src/components/Pivot/PivotItem.types.ts

```diff
@@ -43,7 +43,7 @@ export interface IPivotItemProps extends React.HTMLAttributes<HTMLDivElement> {
    *
    * Example: completed(4)
```

Comment:
While you're editing this property, would you mind updating the doc comment as follows?
```
  /**
   * An optional item count displayed in parentheses just after the `linkText`.
   *
   * Example: completed (4)
   */
```

Reply:
Actually, I think it would be even better something like

```ts
/**
 * Defines an optional item count displayed in parentheses just after the `linkText`.
 *
 * Example: completed (4)
 */
```

Just to be inline with how we're planning to comment our code moving forward. More about those conventions can be found [here](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Commenting-conventions).

Reply:
Sure, I can update that too.

Reply:
How about this?

/**
   * Defines an optional item count displayed in parentheses just after the `linkText`.
   * 
   * Examples: completed (4), Unread (99+)
   */
  itemCount?: number | string;

Reply:
That sounds good to me.